src/sentry/types/issues.py

```diff
@@ -11,6 +11,7 @@ class GroupType(Enum):
     PERFORMANCE_RENDER_BLOCKING_ASSET_SPAN = 1004
     PERFORMANCE_DUPLICATE_SPANS = 1005
     PERFORMANCE_N_PLUS_ONE_DB_QUERIES = 1006
+    PERORMANCE_CONSECUTIVE_DB_OP = 1007
```

Comment:
Typo (should be `PERFORMANCE_...`)

Reply:
Shoot! didn't notice that haha, ill make the change quick!